### PR TITLE
Feature/add to home prompt

### PR DESCRIPTION
**Pull request by thayannevls**
#### What is the purpose of this pull request?
Add InstallBanner to orderPlaced, to show the install prompt.

#### What problem is this solving?
In the store settings, you can choose to show the [install prompt](https://developers.google.com/web/fundamentals/app-install-banners/promoting-install-mobile) when the first purchase is maded. 

#### How should this be manually tested?
[Worskpace](https://orderprompt--storecomponents.myvtex.com/checkout/orderPlaced/?og=949500478939)

#### Screenshots or example usage
Screenshot from 2019-08-13 14-00-57

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/InstallBanner/index.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+interface Props {
+  onInstall: () => any,
+  onDismiss: () => any
+}
+
+const InstallBanner: FunctionComponent<Props> = ({ onInstall, onDismiss }) => {
+  return (
+    <div className="pa6 br2 bg-muted-5 mb8 mw5 w-80-ns w-90 center fr-l">
+        <p className="t-body">
+          <FormattedMessage id="store/install-banner.title"/>
```
Comment (fanny) [inline react/components/InstallBanner/index.tsx]: Use `defineMessages`

File: react/components/OrderInfo/index.tsx [hunk 2]
```diff
@@ -38,7 +38,7 @@ const OrderInfo: FunctionComponent<Props> = ({
 
   return (
     <section>
-      <div className="mv6 w-80-ns w-90 center">
+      <div>
```
Comment (fanny) [inline react/components/OrderInfo/index.tsx]: Useful?

Comment (thayannevls) [reply react/components/OrderInfo/index.tsx]: I had to move this to the `<main>` in order placed, to keep the **banner** aligned as the others elements

**Comment by fanny** [comment]
Add `yarn-error` to gitignore

**Comment by fanny** [comment]
@thayannevls do you made the changes in style?

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+
+- Install Banner PWA.
```
Comment (salesfelipe) [inline CHANGELOG.md]: Shouldn't this be only visible on mobile? Also, shouldn't this be at the top? cc: @alinevillaca

Comment (salesfelipe) [reply CHANGELOG.md]: Maybe this should be a separated app that can be used through out the store. Think about it...

Comment (thayannevls) [reply CHANGELOG.md]: @salesfelipe Maybe I'll rewrite this to be a mini info bar at the top. What do you think?

Comment (salesfelipe) [reply CHANGELOG.md]: The location should be defined by the designers. Talk to @alinevillaca and see what she thinks...

**Pull request closed without merging**